README.md

```diff
@@ -1 +1,29 @@
-# hw01
\ No newline at end of file
+# Welcome ! 
+My name is **Mira** Joseph and I'm a second year Sociology/Literature student in the College. 
+
+### Interests
+
+I am interested in urban walkability, cool maps, Frederic Nietzsche, and Marx; for my BA in  sociology I am considering specializing in either social stratification or the sociology of disaster. During this period of quarantine, I am at home in ~~the greater~~ Boston ~~area.~~
+
+### What I Do
+
+In my free time I enjoy
+
+* [random wikipedia page generator](https://en.wikipedia.org/wiki/Special:Random)
```

Comment:
Your hyperlink text could be describing what you do (to match the second bullet point style) - but the link is great!